rfcs/0000-c-unwind-and-project-group-charter.md

```diff
@@ -6,146 +6,152 @@
+> code faults. Whether the Wasm frames get unwound or just dropped is irrelevant
+> at this point, because as Niko mentioned there are no destructors on those
+> frames, just enough call frame information to let the unwinder do its thing.
 
 Moreover, Lucet executes only on a narrow range of platforms:
 
 * XXX details
 
 ### mozjpeg: XXX
 
-The `mozjpeg-rust` crate provides an idiomatic Rust wrapper over the `libjpeg` library. When this library errors, it calls a user-provided callback that is not allowed to return. The documentation mentions that this callback either aborts or `longjmp` somewhere. The `mozjpeg-rust` crates implements this callback to `panic!` instead, unwinding from the Rust callback into C, and then from C back into Rust.
+The `mozjpeg-rust` crate provides an idiomatic Rust wrapper over the `libjpeg`
+library. When this library errors, it calls a user-provided callback that is
+not allowed to return. The documentation mentions that this callback either
+aborts or `longjmp` somewhere. The `mozjpeg-rust` crates implements this
+callback to `panic!` instead, unwinding from the Rust callback into C, and then
+from C back into Rust.
+
+> XXX this use-case also does involve foreign frames with destructors.
 
 ### lua bindings and longjmp
 
-Although longjmp/setjmp are not directly in scope for this group,
-there is an interesting interaction that was uncovered and is worth
-documenting. On Windows targets specifically, longjmp is implemented
-using the same "Structured Exception Handling" (SEH) mechanism that is
-used for C++ exceptions and other sorts of errors. As a result, in
-some versions of Rust at least, we found that attempts to longjmp over
-Rust frames triggered unwinding mechanisms. Note that in general a
-longjmp over a Rust frame that contains destructors is [Undefined
-Behavior] -- however, we would like to support longjmp for Rust frames
-that do not contain destructors. At minimum, this group should take
-the behavior of longjmp into account; if it is easy to do, we may also
-introduce mechanisms or RFCs that help to specify the use of longjmp
-in particular cases.
+Although `longjmp`/`setjmp` are not directly in scope for this group, there is an
+interesting interaction that was uncovered and is worth documenting. On Windows
+targets specifically, longjmp is implemented using the same "Structured
+Exception Handling" (SEH) mechanism that is used for C++ exceptions and other
+sorts of errors. As a result, in some versions of Rust at least, we found that
+attempts to longjmp over Rust frames triggered unwinding mechanisms. Note that
+in general a longjmp over a Rust frame that contains destructors is [Undefined
+Behavior] -- however, we would like to support longjmp for Rust frames that do
+not contain destructors. At minimum, this group should take the behavior of
+longjmp into account; if it is easy to do, we may also introduce mechanisms or
+RFCs that help to specify the use of longjmp in particular cases.
```

Comment:
This is mostly a change in where I put my newlines. Sorry, I typically use 80-column lines and aggressively use Vim's `gq` to realign! I can reformat to minimize the diff if you'd like.